core/src/test/scala/unit/kafka/admin/DescribeConsumerGroupTest.scala

```diff
@@ -186,6 +186,13 @@ class DescribeConsumerGroupTest extends KafkaServerTestHarness {
       state == Some("Stable")
     }, "Expected the group to initially become stable.")
 
+    // Group assignments in describeGroup rely on finding committed consumer offsets.
+    // Wait for an offset commit before shutting down the group executor.
+    TestUtils.waitUntilTrue(() => {
+      val (_, assignments) = consumerGroupService.describeGroup()
+      !assignments.toSeq.flatMap(_.filter(_.group == group)).isEmpty
```

Comment:
I think you can say something along the lines of `assignments.exists(_.exists(_.group == group))` (I didn't try to compile it).